tests/wicked/startandstop/sut/t12_vlan_xml_config.pm

```diff
@@ -18,17 +18,16 @@ use strict;
 use warnings;
 use testapi;
 use network_utils qw(iface ifc_exists);
+use utils 'file_content_replace';
 
 sub run {
     my ($self) = @_;
+    my $iface  = iface();
     my $config = '/etc/wicked/ifconfig/vlan.xml';
     $self->get_from_data('wicked/xml/vlan.xml', $config);
     my $local_ip = $self->get_ip(type => 'vlan', netmask => 1);
     $local_ip =~ s'/'\\/';
-    my $iface = iface();
-    assert_script_run("sed 's/ip_address/$local_ip/' -i $config");
-    assert_script_run("sed 's/iface/$iface/' -i $config");
-    script_run("cat $config");
+    file_content_replace($config, iface => $iface, ip_address => $local_ip);
```

Comment:
ok to remove here `cat $config` because we don't do it in all the other places. But would it make sense to upload the resulting file as a log with upload_logs() for post-troubleshooting?

Reply:
I would like it. But what happen if we change a config file twice during one test?

Reply:
> I would like it. But what happen if we change a config file twice during one test?

In that case it would be uploaded twice, with different names. We could add a parameter `upload_file => true` in the function, but maybe not for this PR.

Reply:
I would like to keep it , the fact that we haven't done it before and now we doing this I treat as improvement. regarding to upload it with logs , well it could fail in many ways ...